shell/platform/darwin/ios/framework/Source/FlutterEngineTest.mm

```diff
@@ -134,4 +134,28 @@ - (void)testWaitForFirstFrameTimeout {
   [self waitForExpectationsWithTimeout:1 handler:nil];
 }
 
+- (void)testSpawn {
+  FlutterEngine* engine = [[FlutterEngine alloc] initWithName:@"foobar"];
+  [engine run];
+  FlutterEngine* spawn = [engine spawnWithEntrypoint:nil libraryURI:nil];
+  XCTAssertNotNil(spawn);
+}
```

Comment:
check that calling spawn accidentally before run doesn't crash or do something too weird

Reply:
It should crash which is typical for other methods that require an engine to have been run first.